### PR TITLE
Router

### PR DESCRIPTION
**Pull request by ioilmio**
This milestone include:
- routing
- testing

### CODE REVIEW

**Comment by mikenath223** [comment]
### STATUS: INVALID PR ❌

This PR has been marked as invalid.

Reason:
- Please fix the stylelint issues before resubmitting.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @ioilmio 👋

**Nice work here. You did well, adding snapshots on components. 👍**

However, let's some fixes.

- Please refer to comments on the code

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/logic/calculate.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-expressions */
```
Comment (mikenath223) [inline src/logic/calculate.js:1]: - [ ] Kindly ensure that all functions in the calculate.js file are tested with appropriate unit tests.

**Comment by mikenath223** [comment]
- [ ] In the readme the section titled: `Make you operation as usual` isn't related to this project either update it or remove it totally.

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 

Great job @ioilmio, and thanks for making the required changes :hammer: :wrench:.

Now you can merge, your project is approved :checkered_flag: 

## Happy Coding :tophat:

**Pull request merged**